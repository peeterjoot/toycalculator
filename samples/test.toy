DCL x;
DCL y;
x = 5 + 3;
y = x * 2;
PRINT x;
