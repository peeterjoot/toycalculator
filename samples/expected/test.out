8.140000
16.280000
